src/main/java/com/cegeka/academy/service/dto/ChallengeForUserChallengeDTO.java

```diff
@@ -0,0 +1,66 @@
+package com.cegeka.academy.service.dto;
+
+import java.util.Date;
+
+public class ChallengeForUserChallengeDTO {
```

Comment:
Denumirile sunt cam mari. Poti sa le lasi UserChallengeDTO. Ideea este, ca altcineva sa le foloseasca daca se poate acest lucru.

Reply:
Cum ar trebui sa o numesc avand in vedere ca este clasa Challenge care va fi folosita pentru UserChallenge?